### PR TITLE
[OPS-7323] docstore attachment handling

### PR DESCRIPTION
**Pull request by cafuego**

### CODE REVIEW

File: docker/etc/nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  gzip_disable msie6;
+
+  ## Hide the Nginx version number.
+  server_tokens off;
+
+  ## Excluded SSL config.
+
+  ## DNS resolver.
+  ## We use the docker one, which points at the AWS one, so we get internal resolution. This is so yuck.
+  resolver 127.0.0.11;
```
Comment (cafuego) [inline docker/etc/nginx/nginx.conf:112]: 😭

**Review by orakili — COMMENTED** [review]
I could get this to work by putting `resolver 127.0.0.11 ipv6=off;` before the `proxy_pass` and letting the main resolver in `nginx.conf` to `8.8.8.8`:

```
    ## Pass requests for attachments to the docstore.
     location ~ "^/attachments/(?<uuid>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/(?<filename>[^/]+)(?<extension>\.[0-9a-zA-Z]+)$" {
       resolver 127.0.0.11 ipv6=off;
       proxy_pass http://docstore.ahconu.org.internal/files/$uuid/$filename$extension;
     }
```

Note: this didn't work on my setup without the `ipv6=off` whether `resolver 127.0.0.11` was in nginx.conf or in the /files/xx location.

**Comment by cafuego** [comment]
I'd rather not have us use the google resolver at all in our stacks. That was a hack to not be reliant on Contegix DNS. Generally, proxy backends are addressed by IP anyway, but the load balancer has a name, so I can do vhost matching there. For the stack on dev-001, it needs to resolve this name as a docker alias.

File: docker/etc/nginx/nginx.conf (hunk 1 above)
Comment (cafuego) [inline docker/etc/nginx/nginx.conf:112]:
```suggestion
  resolver 127.0.0.11 ipv6=off;
```

**Pull request merged**